### ISSUE TITLE
Variables not set for includes

### ISSUE THREAD

**Issue by bretmartin** [ansible/ansible#11879]
Testing on e16b24c01e, I'm unable to set variables via includes:

main.yml:

```

---

- include: otherfile.yml message=hello
```

otherfile.yml:

```

---

- debug: var=message
```

Results with e16b24c01e:

```
PLAY ****************************************************************************

TASK [setup] ********************************************************************
ok: [localhost]

TASK [test : include message=hello] *********************************************
included: otherfile.yml for localhost

TASK [test : debug var=message] *************************************************
ok: [localhost] => {
    "changed": false,
    "message": "VARIABLE IS NOT DEFINED!"
}

PLAY RECAP **********************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=0
```

Results with tag `v1.9.2-1`:

```
PLAY [localhost] **************************************************************

GATHERING FACTS ***************************************************************
ok: [localhost]

TASK: [test | debug var=message] **********************************************
ok: [localhost] => {
    "var": {
        "message": "hello"
    }
}

PLAY RECAP ********************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=0
```

**Comment by bretmartin** [comment]
Looks like the fix for #11882 (5eb092b) also corrects this. @jimi-c, would you like to confirm?

**Comment by jimi-c** [comment]
Hi @bretmartin yes it looks like the above patch fixed this as well, so closing now.

Thanks!
